website/content/api-docs/agent/index.mdx

```diff
@@ -710,6 +710,14 @@ The table below shows this endpoint's support for
 
 - `node` `(string: <required>)` - Specifies the name of the node to be forced into `left` state. This is specified as part of the URL.
 
+- `prune` `(bool: false)` - Specifies whether to forcibly remove the node from
+  the LAN and WAN gossip pools. This is specified as part of the URL as
+  a query parameter. Added in Consul 1.6.2.
```

Comment:
The `wan` and `prune` flags can be used together. If the `wan` flag is passed with `prune` we force-leave and prune from the WAN pool only.

What if this comment about removing from the LAN and WAN gossip pools was moved to the docs for the endpoint above, rather than next to the prune flag?

```suggestion
- `prune` `(bool: false)` - Specifies whether to forcibly remove the node from the list of members.
  Pruning a node in the `left` or `failed` state removes it from the list altogether. 
  This is specified as part of the URL as a query parameter. Added in Consul 1.6.2.
```